Doc/library/asyncio-eventloop.rst

```diff
@@ -1593,6 +1593,9 @@ Do not instantiate the :class:`Server` class directly.
    .. versionchanged:: 3.7
       Server object is an asynchronous context manager since Python 3.7.
 
+   .. versionchanged:: 3.11
+      :class:`base_events.Server` was exposed as :class:`Server` since Python 3.9.11 3.10.3 and 3.11.
```

Comment:
The `base_events` module isn't mentioned anywhere else in the docs for `asyncio` currently, so I'd just say:

```suggestion
      This class was exposed publicly as ``asyncio.Server`` in Python 3.9.11, 3.10.3 and 3.11.
```